app/scripts/dimApp.i18n.js

```diff
@@ -545,39 +545,197 @@
+            Exclude: "Objetos excluidos",
+            ExcludeHelp: "Shift + click en un objeto (o arrástralo y suéltalo en esta cubeta) para crear un conjunto sin equipo específico.",
+            LockPerk: "Bloquear mejora",
+            Missing1: "Faltan piezas legendarias o exóticas para crear un conjunto completo!",
+            Missing2: "Faltan piezas raras, legendarias o exóticas para crear un conjunto completo!" },
           Loadouts: {
-            Create: "Crear Loadout",
+            Loadouts: "Equipos",
+            Before: "Antes '{{name}}'",
+            Create: "Crear Equipo",
             FromEquipped: "Equipado",
-            Edit: "Editar Loadout",
-            Delete: "Borrar Loadout",
-            ApplySearch: "Artículos = \"{{query}}\"",
-            MaximizeLight: "Maximizar la Luz",
-            ItemLeveling: "Nivelación Elemento",
-            GatherEngrams: "Recopilar Engramas",
-            GatherEngramsExceptExotics: "Exóticas",
-            RestoreAllItems: "Todos los Artículos",
-            Loadouts: "Loadouts" },
+            Edit: "Editar equipo",
+            Delete: "Borrar equipo",
+            ConfirmDelete: "Estas seguro de que quieres borrar '{{name}}'?",
+            ApplySearch: "Objetos = \"{{query}}\"",
+            MaximizeLight: "Maximizar Luz",
+            ItemLeveling: "Nivel de objetos",
+            GatherEngrams: "Juntar engramas",
+            GatherEngramsExceptExotics: "Exóticos",
+            RestoreAllItems: "Todos los objetos",
+            LoadoutName: "Nombre de equipo...",
+            Save: "Guardar",
+            SaveAsNew: "Guardar como nuevo",
+            Cancel: "Cancelar",
+            ItemsWithIcon: "Objetos con éste ícono van a ser equipados.",
+            ClickToEquip: "Click en un objeto para equipar.",
+            AppliedAuto: "Crear automático de equipo",
+            Applied: "Tu equipo de {{amount}} objetos han sido transferidos a tu {{store}}.",
+            AppliedError: "Ninguno de los objetos en tu equipo pudo ser transferido.",
+            AppliedWarn: "Tu equipo ha sido parcialmente transferido, pero {{failed}} de {{total}} objetos tuvieron errores." },
+          Manifest: {
+            Build: "Creando información de la base de datos de Destiny",
+            Download: "Descargando información más actualizada de Destiny desde Bungie",
+            Error: "Error cargando Destiny info:\n{{error}}\n Carga de nuevo para volver a intentarlo.",
+            Outdated: "Información de Destiny obsoleta",
+            OutdatedExplanation: "Bungie ha actualizado su información de base de datos de Destiny. Recarga DIM para cargar la información nueva. Toma en cuenta que puede que DIM no funcione por algunas horas después de que Bungie actualiza Destiny, mientras que los nuevos datos se propagan por sus sistemas.",
+            BungieDown: "Bungie.net puede estar teniendo problemas.",
+            Load: "Cargando información de Destiny guardada",
```

Comment:
Can you please add LoadCharInv: "Loading Destiny characters and inventory", here with the correct translation, do not add it to the English definitions though, it is a new translation that will automatically be applied at a later date.